packages/connect-form/src/utils/connect-form-errors.ts

```diff
@@ -13,15 +14,25 @@ interface HostFieldError {
   message: string;
 }
 
-interface SchemaFieldError {
+export interface SchemaFieldError {
   fieldName: MARKABLE_FORM_FIELD_NAMES.IS_SRV;
   message: string;
 }
 
+export interface SSHTunnelFieldError {
+  fieldName: MARKABLE_FORM_FIELD_NAMES.IS_SSH;
+  errors: SSHFormErrors;
+}
+
+export type SSHFormErrors = {
+  [key in keyof SSHConnectionOptions]?: string;
```

Comment:
This is inspiring me to lean very deep into and learn all the latest and greatest new syntax :)